### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by Nekkok** [BelgianSalamander/MoreBundles#20]
Suddenly, my game is crashing after testing it with More Bundles since the beggining of its creation. I'm afraid that this won't get resolved since it's been a while since last update, so please at least if you could provide some solution for it it would be wonderful <3
[latest.log](https://github.com/BelgianSalamander/MoreBundles/files/11215117/latest.log)
[crash-2023-04-12_21.16.48-client.txt](https://github.com/BelgianSalamander/MoreBundles/files/11215118/crash-2023-04-12_21.16.48-client.txt)
[config.txt (converted for uploading directly)](https://github.com/BelgianSalamander/MoreBundles/files/11215124/config.txt)

**Comment by Nekkok** [comment]
So, after deleting the morebundles config folder, the first try running it after each run it loads without problem but then it crashes. I don't know what could be causing this issue after loading the default configs...

**Comment by BelgianSalamander** [comment]
I think the problem with your original config is that in newer versions I removed the "WARN" (and also "IGNORE") error policies as they lead to too many issues (although the wiki doesn't reflect that change - most of the info on the config is out of date and I intend on updating it at some point). Instead try the "FAIL ON ERROR" policy.

As for it crashing with the default config, I'm not sure what could be causing that. Could you provide more details? Thanks :)

**Comment by Nekkok** [comment]
Forgive me I couldn't reply earlier! I've been trying to mess with the configs in many different approaches, but at the end I get this message in the crash report. Also sending you the config again:
[config.txt](https://github.com/BelgianSalamander/MoreBundles/files/11226117/config.txt)
[crash-2023-04-13_22.07.09-client.txt](https://github.com/BelgianSalamander/MoreBundles/files/11226112/crash-2023-04-13_22.07.09-client.txt)

**Comment by Nekkok** [comment]
And yes, I'm sending you the config tweaked to my tastes, as I'm using your mod for a pack where I want to use the custom tooltip when you hover the bundle (it is perfect I must say), then this is the crash I get. Also, from my previous log, you can see what other mods I'm using.
[config.txt](https://github.com/BelgianSalamander/MoreBundles/files/11226203/config.txt)
[crash-2023-04-13_22.22.36-client.txt](https://github.com/BelgianSalamander/MoreBundles/files/11226204/crash-2023-04-13_22.22.36-client.txt)

**Comment by Nekkok** [comment]
And last but not least, this are my configs and mods I was using in a previous version of the pack everything was working fine. Hope all this info help you to spot any problems:
[config.txt](https://github.com/BelgianSalamander/MoreBundles/files/11226216/config.txt)
[mods.txt](https://github.com/BelgianSalamander/MoreBundles/files/11226224/mods.txt)

**Comment by BelgianSalamander** [comment]
For the first crash I'm not exactly sure what's causing it. It seems either minecraft's bundle texture or my base bundle textures are somehow getting corrupted? I have attached a build of the mod which should hopefully circumvent the issue. Please test it if you have the time and if it does work I will upload it to curseforge.

As for the second crash, it looks like you added a file directly into the resources/assets or resources/data folder of the config without placing it in the proper namespace folder and the code panics because it can't extract the namespace from that. Admittedly, this probably shouldn't cause a crash so this should also be fixed in the jar attached. If you don't understand what I mean by this or need help fixing this I'd be happy to help.

If you do have issues with this new jar, it might also be helpful if you also send a full log through, as this could shine more light on what is happening in the first crash if what I have done hasn't fixed it.

Thanks a lot for your help :)
[morebundles-fabric-1.19.2-0.3.0.zip](https://github.com/BelgianSalamander/MoreBundles/files/11228655/morebundles-fabric-1.19.2-0.3.0.zip) (I have attached the file as a zip because github doesn't allow me to upload jars directly)

**Comment by Nekkok** [comment]
Well, you made magic sir. Works like a charm! I don't think I changed anything in the assets folder, but if the log says it... It's just I can't remember, the only thing I did was to delete the generated data folder to remove the recipes, but I will leave it as it is and I will try to change the recipe then.

But to the point, the jar works well. I couldn't load the game after the first run and now everything is fine. And to show my genuine enthusiasm (or annoyance, like you want to take it), will you update your mod for 1.20 when it comes out? Now that Mojang "somehow" has implemented the bundle, your mod is like a pretty needed upgrade for the capabilities it can offer.

Thank you so much for your assistance : )
